### ISSUE TITLE
Wood White being classified as a non-butterfly

### ISSUE THREAD

**Issue by IanMiddlebrook** [BiologicalRecordsCentre/UKBMS-online#242]
Hi @Gary-van-Breda 

One of our recorders (username 'J+C Taylor') has been entering data for Grafton Wood transect. This week he found that Wood White was no longer available in the data entry page for butterflies, but found it among 'other taxa'. When he looks at the Annual Summary page, he finds that the Wood White recorded on August 22nd is not displayed unless he checks to display non-butterfly species (although previous Wood White are all fine). This latest Wood White is also not included in the standard downloads unless non-butterfly species are included.

Thanks, Ian

**Comment by Gary-van-Breda** [comment]
Hi @IanMiddlebrook
When the occurrence was recorded in the "Other taxa",  it got recorded against the Wood White in the UK Master List, not the UKBMS Butterfly list: these have different taxon ids (but the same meaning).
This is the only place this has happened for Wood White across the entire Data set.  
I've also checked that Wood White now appears correctly on the Butterflies tab autocomplete (when the fix for #260 is included).  
I have reassigned the occurrence to the correct taxon ID, and re-run the walk through the summary builder. The August 22 Wood white is now being shown as a butterfly, rather than non-butterfly.

For future info:
UK Master List (15): TTL ID 151131
UKBMS Butterfly List (51): TTL ID 283038
Aug 22 Grafton Wood sample_id=15402017
Wood White occurrence_id= 21735324
